<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPasswordWindow</class>
 <widget class="QWidget" name="SetPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Придумайте пароль</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {font-family: Yu Gothic UI Semibold;
font-size: 19px;}</string>
   </property>
   <property name="text">
    <string>Придумайте пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Yu Gothic UI Semibold;
font-size: 26px;</string>
   </property>
   <property name="text">
    <string>Придумайте пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_password_line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="set_password_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>241</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {font-family:Yu Gothic UI ;
					font-size: 20px;
					background-color: #FFF44F;
					border: 1px solid #911E42;
					border-radius: 5px;
					color: #911E42;
					position: relative;
					box-sizing: border-box;
					transition: all 700ms ease;}

QPushButton:hover {
	background: #911E42;
	color: #FFF44F;
	box-shadow: inset 0 0 0 3px #b84642;
}</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {font-family: Yu Gothic UI Semibold;
font-size: 19px;}</string>
   </property>
   <property name="text">
    <string>Повторите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeat_password_line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>203</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
